attest_fn_call/README.md

```diff
@@ -0,0 +1,6 @@
+# Attesting a Function Call
+
+For a walkthrough of this example, check out its 
```

Comment:
The goal is for the examples to be only code, with all the documentation being available through the documentation site